### ISSUE TITLE
OpenEye reoders atoms when writing to PDB

### ISSUE THREAD

**Issue by jthorton** [openforcefield/openforcefield#673]
**Describe the bug**
I think when using openeye to write a molecule to PDB file through the toolkit it puts the molecule in canonical order meaning the ordering is not consistent with the Molecule object. This stops when using other files types like SDF.

**To Reproduce**
```python
from openforcefield.utils.toolkits import OpenEyeToolkitWrapper, RDKitToolkitWrapper
from openforcefield.topology import Molecule
mol = Molecule.from_smiles("OCCO")
mol.generate_conformers()
mol
mol.to_file(file_path="openeye.pdb", file_format="pdb", toolkit_registry=OpenEyeToolkitWrapper())
mol.to_file(file_path="rdkit.pdb", file_format="pdb", toolkit_registry=RDKitToolkitWrapper())
open_mol = Molecule.from_file("openeye.pdb")
rdkit_mol = Molecule.from_file("rdkit.pdb")
iso, atom_map = Molecule.are_isomorphic(open_mol, rdkit_mol, True)
print(atom_map)
{0: 1, 1: 0, 2: 2, 3: 3, 4: 4, 5: 9, 6: 5, 7: 6, 8: 7, 9: 8}
```

**Output**
The full error message (may be large, that's fine. Better to paste too much than too little.)

**Computing environment (please complete the following information):**
Packages
openeye-toolkits          2020.0.4                 py37_0    openeye
openforcefield            0.7.1              pyh39e3cac_0    omnia
rdkit                     2020.03.4        py37h585bbb4_0    conda-forge

**Additional context**

**Comment by jchodera** [comment]
The [default PDB flavor](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemConstants/OEOFlavor.html#OEChem::OEOFlavor::PDB) includes [`OEOFlavor_PDB_OrderAtoms`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemConstants/OEOFlavor.html#OEChem::OEOFlavor::PDB::OrderAtoms), which might be responsible for this behavior. If we instead feed the writer a specific flavor that omits this component of the favor, I think this might fix it. If not (or I'm misunderstanding this option), I bet we could get OpenEye to implement a flavor flag to preserve atom ordering.

**Comment by j-wags** [comment]
Whoops -- Forgot to include this in Pavan's recent PR. This was closed by #715 and the fix will come out in the 0.8.0 release.
